xonsh/environ.py

```diff
@@ -975,7 +975,10 @@ def DEFAULT_DOCS():
             configurable=ON_WINDOWS,
         ),
         "LANG": VarDocs("Fallback locale setting for systems where it matters"),
-        "LS_COLORS": VarDocs("Color settings for ``ls`` command line utility"),
+        "LS_COLORS": VarDocs(
+            "Color settings for ``ls`` command line utility",
+            default="``" + LsColors(LsColors.default_settings).detype() + "``",
```

Comment:
It isn't great that this will execute on import, effectively.  Is there any way this could just be a static string?

Reply:
Thanks for reviewing this!
The reason I used this method is to avoid peeling between documents and the real default value and make it easy to maintenance documents. But I agree that executing on import is not good. I will try to fix this.